docs/plugins/filters/grok.asciidoc

```diff
@@ -339,25 +306,16 @@ necessarily need to define this yourself unless you are adding additional
 patterns. You can point to multiple pattern directories using this setting.
 Note that Grok will read all files in the directory matching the patterns_files_glob
 and assume it's a pattern file (including any tilde backup files). 
-
 [source,ruby]
------
     patterns_dir => ["/opt/logstash/patterns", "/opt/logstash/extra_patterns"]
------
 
 Pattern files are plain text with format:
-
 [source,ruby]
------
     NAME PATTERN
------
 
 For example:
-
 [source,ruby]
------
     NUMBER \d+
------
 
```

Comment:
I added more formatting in logstash-docs purely for consistency, hence the mismatch with source. This works, and I can live with it.